idaes/ui/flowsheet.py

```diff
@@ -259,6 +260,9 @@ def _map_edges(self):
         # Note we're only using the keys from self.ports, {port: parentcomponent}
         return set(self.ports) - used_ports
 
+    def _convert_for_ajax_compatibility(self, df):
```

Comment:
I would name this more like "_make_valid_json" since AJAX is an implementation detail. Also a docstring saying what it's doing?

Reply:
That makes more sense.

Changed the name and added a docstring